### PR TITLE
Feature/979 reviews sidebar

### PR DESCRIPTION
**Pull request by chris-lawton**
- Change reviews sidebar to use grid instead of a table
- Update filenames accordingly
- Adds styles for agree/disagree reviews

Previously reviewed here - https://github.com/OpenTechFund/opentech.fund/pull/1025

### CODE REVIEW

**Comment by chris-lawton** [comment]
@emullaney - this contains the styling work for the agree/disagree block, branched from your work on it

File: opentech/apply/funds/templates/funds/applicationsubmission_admin_detail.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <div class="sidebar__inner">
         <h5>Reviews &amp; assignees</h5>
 
-        {% include 'funds/includes/review_table.html' %}
+        {% include 'funds/includes/review_sidebar.html' %}
```
Comment (emullaney) [inline opentech/apply/funds/templates/funds/applicationsubmission_admin_detail.html]: great idea to rename this

File: opentech/apply/funds/templates/funds/includes/review_sidebar.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                {% include 'funds/includes/review_sidebar_item.html' with review=review_data.review reviewer=review_data.reviewer opinions=review_data.opinions %}
+            {% endfor %}
+
+            {% for review_data in reviews_block.external_not_reviewed %}
+                {% include 'funds/includes/review_sidebar_item.html' with reviewer=review_data.reviewer missing=True class="hidden"  %}
+            {% endfor %}
+
+            <li><a class="link link--bold link--underlined js-toggle-reviewers" href="#">All Assigned Advisors</a></li>
+        {% endif %}
+    {% endif %}
```
Comment (emullaney) [inline opentech/apply/funds/templates/funds/includes/review_sidebar.html]: This was probably my fault in the original file that this was copied from, but these two `{% endif %}` tags are indented incorrectly

Comment (chris-lawton) [reply opentech/apply/funds/templates/funds/includes/review_sidebar.html]: good spot! will update

**Pull request merged**